s2a/src/main/java/io/grpc/s2a/S2AChannelCredentials.java

```diff
@@ -58,14 +57,12 @@ public static Builder newBuilder(String s2aAddress, ChannelCredentials s2aChanne
   @NotThreadSafe
   public static final class Builder {
     private final String s2aAddress;
-    private final ChannelCredentials s2aChannelCredentials;
-    private ObjectPool<Channel> s2aChannelPool;
+    private ChannelCredentials s2aChannelCredentials;
```

Comment:
Add back `final`? (Seems it may have been a bad merge.)

Reply:
Yup, it was a bad merge, fixed.